src/app/SIPUserAgents/SIPUserAgent.cs

```diff
@@ -1449,6 +1449,16 @@ private async void ClientCallAnsweredHandler(ISIPClientUserAgent uac, SIPRespons
         {
             if (sipResponse.StatusCode >= 200 && sipResponse.StatusCode <= 299)
             {
+                if (sipResponse.Body == null && ((MediaSession as RTPSession)?.IsStarted ?? false))
+                {
+                    m_sipDialogue = uac.SIPDialogue;
+                    m_sipDialogue.DialogueState = SIPDialogueStateEnum.Confirmed;
+
+                    logger.LogInformation($"Call attempt to {m_uac.CallDescriptor.Uri} was answered; no media update from early media.");
+
+                    ClientCallAnswered?.Invoke(uac, sipResponse);
+                    return;
+                }

```

Comment:
This looks to be a good pragmatic approach. I'd still like to find where in an RFC it states this approach can be used so I can add a comment but in the meantime I can't see a reason not to do it like this.